index.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <title>Developer's Summit</title>
+  </head>
+  <body>
+    <!--Navbar-->
+    <nav>
+      <div class="logo-container">
+        <a href="index.html" class="logo">
+          <span class="highlight">Dev</span>Summit<span class="highlight"
+            >.</span
+          >
+        </a>
+      </div>
+      <div class="nav-options">
+        <ul class="menu">
+          <li class="menu-item"><a href="about.html">About</a></li>
+          <li class="menu-item"><a href="#">Program</a></li>
+          <li class="menu-item"><a href="#">Join</a></li>
+          <li class="menu-item"><a href="#">Sponsor</a></li>
+          <li class="menu-item"><a href="#">News</a></li>
```

Comment:
- [ ] kindly add links to the nave sections. they should take you to their respective sections